bot/cogs/git.py

```diff
@@ -131,6 +131,30 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    
+    @git.command(help='Prints info on the git pull command')
+    async def pull(self, ctx):
+        msg = ManMessage(
+            name='git-pull'
```

Comment:
```suggestion
            name='git-pull',
```
There is a missing comma here.